src/components/Form/form.ts

```diff
@@ -0,0 +1,66 @@
+import Block from '../../modules/Block';
+import {FormProps} from './types';
+import {template} from './template';
+import Input from '../Input';
+import {TRenderElement} from '../../modules/Block/types';
+import {InputName} from '../../shared/const';
+
+class Form extends Block<FormProps> {
+  constructor(props:FormProps) {
+    super({
+      ...props,
+      events: {
+        submit: (event: Event) => this.onSubmit(event),
```

Comment:
Если не прокидываются доп. аргументы то такие записи можно сократить
```suggestion
        submit: this.onSubmit,
```

Reply:
не заработал у меня сокращенный формат записи, ошибку выдает